core/trino-main/src/main/java/io/trino/execution/scheduler/NodeAssignmentStats.java

```diff
@@ -66,7 +66,12 @@ public long getTotalSplitsWeight(InternalNode node)
 
```

Comment:
nit: commit message "... amount of max pending splits to be assigned is duplicated in the next iteration.." -> " amount of max pending splits to be assigned is **doubled** in the next iteration"